lib/spack/docs/basic_usage.rst

```diff
@@ -1222,11 +1026,11 @@ Module Commands for Shell Scripts
 
 Although Spack is flexible, the ``module`` command is much faster.
 This could become an issue when emitting a series of ``spack load``
-commands inside a shell script.  By adding the ``--shell`` flag,
-``spack module find`` may also be used to generate code that can be
-cut-and-pasted into a shell script.  For example:
+commands inside a shell script.  The ``spack module loads`` may also
+be used to generate code that can be cut-and-pasted into a shell
+script.  For example:
 
-.. code-block:: console
+.. code-block:: sh
```

Comment:
Good catch.

Do you have guidelines on when to use console vs. sh?  I didn't find any
on-line.  Maybe this should go in our Developer's Guide, along side the
coding style guidelines?

-- Elizabeth

On Wed, Aug 31, 2016 at 10:29 AM, Adam J. Stewart <notifications@github.com>
wrote:

> In lib/spack/docs/basic_usage.rst:
>
> >
> > -.. code-block:: console
> > +.. code-block:: sh
>
> This is probably just a rebase mistake, but this code-block needs to be
> formatted as console output and not a bash script.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/LLNL/spack/pull/1676/files/ca957ac13c11c1086abc32c208987984c3fd0f71#r76997927>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1cd0zHmtf5zRWxz9GgVYiHeoMPUC7pks5qlY_JgaJpZM4JxOaa>
> .
>